### PR TITLE
This is in response to a old Feature Request to add a Make -All Menuswapper addition

### PR DESCRIPTION
**Pull request by Pashweetie**
I've just rebased my code and this should be ready to be merged. Really simple addition really. I think this meets our conventions but correct me if I'm wrong.

### CODE REVIEW

**Comment by Pashweetie** [comment]
I don't know how I am supposed to fix this checksytle error because I checked and there are no space indents where travis thinks there are.

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 1]
```diff
@@ -329,6 +329,11 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 		String option = Text.removeTags(event.getOption()).toLowerCase();
 		String target = Text.removeTags(event.getTarget()).toLowerCase();
 
+        if (option.equals("make one") && config.makeAll())
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java]: All of these lines should be indented with tabs, not spaces.

**Comment by kulershk** [comment]
I guess this is one of these things: https://github.com/runelite/runelite/wiki/Rejected-or-Rolled-Back-Features

**Comment by Trevor159** [comment]
This is no longer relevant with the "new" make all item screen

**Pull request closed without merging**